packages/bpk-docs/src/pages/DialogsPage/DialogsPage.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import BpkDialog from 'bpk-component-dialog';
+import BpkButton from 'bpk-component-button';
+import { BpkButtonLink } from 'bpk-component-link';
+
+import dialogReadme from 'bpk-component-dialog/readme.md';
+
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import Paragraph from './../../components/Paragraph';
+
+class DialogContainer extends Component {
+  constructor() {
+    super();
+
+    this.onOpen = this.onOpen.bind(this);
+    this.onClose = this.onClose.bind(this);
+
+    this.state = {
+      isOpen: false,
+    };
+  }
+
+  onOpen() {
+    this.setState({
+      isOpen: true,
+    });
+  }
+
+  onClose() {
+    this.setState({
+      isOpen: false,
+    });
+  }
+
+  render() {
+    const { children, ...rest } = this.props;
+
+    return (
+      <div>
+        <BpkButton onClick={this.onOpen}>Open Dialog</BpkButton>
+        <BpkDialog
+          closeLabel="Close dialog"
+          id="my-dialog"
+          className="my-classname"
+          isOpen={this.state.isOpen}
+          onClose={this.onClose}
+          renderTarget={() => document.getElementById('application-container')}
+          getApplicationElement={() => document.getElementById('portal-target')}
+          {...rest}
+        >
+          <div>{children}</div>
+          <BpkButtonLink onClick={this.onClose}>Close dialog</BpkButtonLink>
+        </BpkDialog>
+      </div>
+    );
+  }
+}
+
+DialogContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default dialog',
+    blurb: [
+      <Paragraph>
+        The default has a title and close icon. Tapping the scrim or close icon
+        will dismiss the dialog.
+      </Paragraph>,
+    ],
+    examples: [
+      <DialogContainer title="Default Dialog.">
+        <Paragraph>You can put anything you want in here.</Paragraph>
+      </DialogContainer>,
+    ],
+  },
+  {
+    id: 'non-dismissible',
+    title: 'Non dismissible',
+    blurb: [
+      <Paragraph>
+        In cases when you want the user to explicitly make or accept the choice,
+        this configuration means the dialog has no close icon, and tapping the
+        scrim will NOT dismiss it. Instead, the user must interact with the
```

Comment:
We shouldn't use `NOT` here, as it isn't good to use uppercase for emphasis. We should instead use bold text.